node-graph/gcore/src/value.rs

```diff
@@ -115,6 +115,7 @@ impl<'i, T: Clone + 'i> Node<'i, ()> for DebugClonedNode<T> {
 }
 
 impl<T: Clone> DebugClonedNode<T> {
+	#[allow(clippy::new_ret_no_self)]
 	pub const fn new(value: T) -> ClonedNode<T> {
```

Comment:
This function should return a `DebugClonedNode`, this is probably just a copy/paste error.